src/ChronosInterface.php

```diff
@@ -534,6 +534,22 @@ public function isLeapYear(): bool;
      */
     public function isSameDay(ChronosInterface $dt): bool;
 
+    /**
+     * Checks if the passed in date is the same month as the instance current month.
+     *
+     * @param \Cake\Chronos\ChronosInterface $dt The instance to check against.
+     * @return bool
+     */
+    public function isSameMonth(ChronosInterface $dt): bool;
```

Comment:
Interfaces can only we changed in major releases, so this needs to target 3.x.

Reply:
Can it be added to 2.x without change of interface?

Reply:
Theoretically it can be but IMO this isn't an important enough addition to deviate from the interface.

Reply:
Ok, but can you help with rebase? For me it gives conflicts in files. Or should create new PR?

Reply:
Creating a new PR would be easier :)